utils/s2n_safety.h

```diff
@@ -100,7 +100,7 @@ static inline void* trace_memcpy_check(void *restrict to, const void *restrict f
 #define GUARD_NONNULL_PTR( x )          do {if ( (x) == NULL ) return NULL;} while (0)
 
 /* Check the return value from caller. If this value is -2, S2N_ERR_BLOCKED is marked*/
```

Comment:
is this still -2?

Reply:
It is still -2. I'm just giving the code a #define to make it more explicit what is happening. Callers should return S2N_ERR_CACHE_WILL_BLOCK.

Reply:
Nope, I take it back. I was still messing with this error code. You are right it needs to remain -2. I'll put that back during today's refactor.